src/backend/tests/it/upgrade/credentials_init_args.rs

```diff
@@ -15,7 +15,10 @@ fn test_upgrade_credential_init_args() {
         allowed_callers: allowed_callers.clone(),
     });
     let encoded_initial_arg = encode_one(initial_arg).unwrap();
-    let pic_setup = setup_with_custom_wasm(BACKEND_V0_0_25_WASM_PATH, Some(encoded_initial_arg));
+    let pic_setup = BackendBuilder::default()
+        .with_wasm(BACKEND_V0_0_25_WASM_PATH)
+        .with_arg(encoded_initial_arg)
```

Comment:
I imagine you already checked, but here it should not be optional, right? `Some(encoded_initial_arg)`